lab_1/task_1/build-client.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+DEFAULT_CONFIGURATION=development
+
+if [ -z "$1" ]; then
+  ENV_CONFIGURATION=$DEFAULT_CONFIGURATION
+else
+  ENV_CONFIGURATION=$1
+fi
+
+if [ -f dist/client-app.zip ]; then
+    rm dist/client-app.zip
+fi
+
+npm i
```

Comment:
If we speak about this script usage cases, there is high possibility that it will be used in CI pipeline. So it's better to use "npm ci" instead of "npm install"